README.md

```diff
@@ -134,6 +134,13 @@ The new hood.ie Website doesn't need to be deployed after making changes. (Only
     - run `bundle exec jekyll serve --watch --drafts` in your terminal and wait until it says `server running`
     - go to `localhost:4000/blog`. All currently available drafts are then just displayed as regular Blog posts. You can now check your draft and edit it in your editor. (Note: Jekyll is sometimes a bit slow, so this may take a little bit.)
 
+### Checking for insensitive content in the post
+* [Alex.js](http://alexjs.com/) is used to check for any insensitve content
+* When you want to check if the draft contains any insensitive content,
+    - run `npm run alex -f _drafts/<your-post>.md`
```

Comment:
I think that should be `npm run alex -- -f _drafts/<your-post>.md`. We also usually don’t use the _drafts folder, we put blog posts into the `_posts` folder directly, so maybe let’s change that to `npm run alex -- -f _posts/<your-post-filename>.md`